pwa/src/styling/design-tokens/conduction-design-tokens.css

```diff
@@ -25,4 +25,7 @@
   --conduction-content-link-font-size: 14px;
 
   --conduction-top-nav-secondary-font-size: 16px;
+
+  --conduction-footer-logo-height-size: 72px;
+  --conduction-footer-font-color: #ababab;
```

Comment:
Because the logo's do not render nicely on the dark background, we're changing the background (see other comments). 

By changing the background, this one also needs changing. I recommend using the `--denhaag-color-grey-4`.